src/test/java/uk/gov/ons/ssdc/caseprocessor/messaging/TelephoneCaptureReceiverIT.java

```diff
@@ -36,10 +37,9 @@
 @ActiveProfiles("test")
 @SpringBootTest
 @ExtendWith(SpringExtension.class)
-public class TelephoneCaptureReceiverIT {
+class TelephoneCaptureReceiverIT {
```

Comment:
class should remain public

Reply:
The linter was complaining at me to scrap them, don't mind changing the linting rules instead if you disagree.

Reply:
hmmm, so it does seem that JUnit5 style is to leave them `protected`

Reply:
tested in IDE and it seems fine. Guess it was JUnit 4 which was more fussy